pkg/network/ebpf/c/prebuilt/tracer.c

```diff
@@ -1,8 +1,8 @@
-#include "kconfig.h"
+//#include "kconfig.h"
```

Comment:
Not needed?

Reply:
Oops. That is my bad. Was debugging an include issue. Forgot to uncomment.